ts/receiver/contentMessage.ts

```diff
@@ -343,11 +345,21 @@ export async function innerHandleSwarmContentMessage(
     const content = SignalService.Content.decode(new Uint8Array(plaintext));
     perfEnd(`SignalService.Content.decode-${envelope.id}`, 'SignalService.Content.decode');
 
-    const blocked = await isBlocked(envelope.source);
+    /**
+     * senderIdentity is set ONLY if that message is a closed group message.
+     * If the current message is a closed group message,
+     * envelope.source is going to be the real sender of that message.
+     *
+     * When receiving a message from a user which we blocked, we need to make let
```

Comment:
* we need to let the control message through

Reply:
updated

Reply:
👍